packages/inventory/src/containers/ServicePointProfile/index.tsx

```diff
@@ -197,6 +197,10 @@ const ServicePointProfile = (props: ServicePointsProfileTypes) => {
     servicePointProfileURL,
   };
 
+  const coordinates = getCords(structure.geometry);
+  const latLong = [coordinates.lat, coordinates.lng];
+  const latLongString = latLong.some((el) => el) ? latLong.join(',') : '';
```

Comment:
Why not just do `latLong.join(',') : '';`

Reply:
@kelvin-muchiri  thought this would be cleaner, removing the change we would endup with the string `, ` as the value to latLongString

Reply:
@p-netm Under what conditions would `,` be displayed?

Reply:
if a location does not have geometry

Reply:
or its illformed, for instance if we get a structure whose geometry type is not Point